tests/core/component.test.js

```diff
@@ -192,4 +192,17 @@ suite('Component', function () {
       assert.deepEqual(componentString, 'position:1 2 3');
     });
   });
+
+  suite('extendSchema', function () {
+    test('the schema is extended', function () {
```

Comment:
`extends the schema` so that the test suite + test reads `extendSchema extends the schema`